sdk/storage/azblob/internal/shared/batch_transfer.go

```diff
@@ -11,10 +11,14 @@ import (
 	"errors"
 )
 
+const (
+	DefaultConcurrency = 5
+)
 // BatchTransferOptions identifies options used by doBatchTransfer.
 type BatchTransferOptions struct {
 	TransferSize  int64
 	ChunkSize     int64
+	NumChunks     uint16
```

Comment:
Why did this become configurable?  It's internal only so no public surface area change, but it looks like in both cases the same value is used.

Reply:
In downloadFile() method, we use `numChunks` to define the length of the array which holds these channels. In BatchTransfer, numChunks is used to split the file into so many chunks. They are the same parameter, and computing it twice (once in downloadFile and once in doBatchTransfer()) may cause inconsistencies.